plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -369,7 +371,10 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
         int debugPort = getDebugPort();
         setupDebugTarget(devServerRunConfiguration, launch, debugPort, monitor);
       }
-      IVMInstall vmInstall = getVMInstall(configuration);
+      
+      IJavaProject javaProject = JavaCore.create(modules[0].getProject());
```

Comment:
I  looked into this. We can cast the IVMInstall to an IVMInstall2 and then use getVersion. Then we have to add code to properly compare java.version strings and find the most recent. Completely doable, but it would take time and IMHO we have higher priorities to focus on right now. If thismulti-module/multi-verison apps prove to be an issue in practice, we can expand this.